frontends/ol-components/src/components/Card/Card.tsx

```diff
@@ -130,17 +139,72 @@ const Bottom = styled.div`
 const Actions = styled.div`
   display: flex;
   gap: 8px;
-  position: absolute;
-  bottom: 16px;
-  right: 16px;
```

Comment:
Previously, we used absolute positioning because:
- The "main" card element was an anchor
- anchors can't have interactive children (invalid HTML)
- so we made anchor + buttons children of a wrapper

Since the main card element is no longer an anchor, this is no longer necessary.